Collapse.php

```diff
@@ -96,7 +96,11 @@ class Collapse extends Widget
      * @since 2.0.7
      */
     public $autoCloseItems = true;
-
+    
+     /**
+     * @var boolean whether head tag converts to 'a' or 'div' that may keep functional label inputs and other link structures 
+     */
+    public $headerLink= true;
```

Comment:
Why div and not another tag?

Reply:
Because default title tags are h groups (h1,h2...) so that div container is valid.  If you use only ‘a’ tag we cant use another ‘a’ tag or input tags in that. My commit is a small solition for this.

Reply:
Well, `<span>` is valid as well.